avaframe/out3Plot/outAB.py

```diff
@@ -244,9 +244,13 @@ def WriteResults(eqOutput, saveOutPath):
     name = eqOutput['Name']
 
     log.info('Profile: %s with %s parameter set',  name, parameterSet)
-    log.info('Alpha point (x,y,z,s) in [m]:(%.2f,%.2f,%.2f,%.2f) and'
-             ' angle in [°] : %.2f' % (x[ids_alpha], y[ids_alpha],
-                                       z[ids_alpha], s[ids_alpha], alpha))
+    if ids_alpha:
+        log.info('Alpha point (x,y,z,s) in [m]:(%.2f,%.2f,%.2f,%.2f) and'
```

Comment:
Use the string multiplier for the format string:
https://stackoverflow.com/a/17278762/13011708